packages/backend/src/controllers/authentication.ts

```diff
@@ -95,7 +103,15 @@ const createOpenIdUserFromUserInfo = (
     issuerType: OpenIdIdentityIssuerType,
     fail: OpenIDClientOktaStrategy['fail'],
 ) => {
-    if (!userInfo.email || !userInfo.sub) {
+    /**
```

Comment:
One of many quirks to account for when using `openid-client` + its built-in strategy + azuread

Reply:
But this method is only used in the okta authentication.

Reply:
Yep - that wasn't the case for a while, will revert these changes to avoid confusion.